include/zephyr/sys/arch_interface.h

```diff
@@ -787,10 +787,12 @@ int arch_buffer_validate(void *addr, size_t size, int write);
  * This call returns the optimal virtual address alignment in order to permit
  * such optimization in the following MMU mapping call.
  *
- * @param[in] phys Physical address of region to be mapped, aligned to MMU_PAGE_SIZE
- * @param[in] size Size of region to be mapped, aligned to MMU_PAGE_SIZE
+ * @param[in] phys Physical address of region to be mapped,
+ *                 aligned to CONFIG_MMU_PAGE_SIZE
+ * @param[in] size Size of region to be mapped,
+ *                 aligned to CONFIG_MMU_PAGE_SIZE
```

Comment:
```suggestion
 * @param[in] phys Physical address of region to be mapped,
 *                 aligned to @kconfig{CONFIG_MMU_PAGE_SIZE}
 * @param[in] size Size of region to be mapped,
 *                 aligned to @kconfig{CONFIG_MMU_PAGE_SIZE}
```

Reply:
Does it generate link to the kconfig doc? My local build doesn't link anything but simply making it with monospace fonts.

Reply:
Yes but only when rendered on the main docs website, not in the Doxygen docs itself

https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/zephyr.doxyfile.in#L278